dataprofiler/data_readers/json_data.py

```diff
@@ -244,15 +244,19 @@ def _load_data_from_file(self, input_file_path):
         :type input_file_path: str
         :return:
         """
-        self._file_encoding = data_utils.detect_file_encoding(input_file_path)
-        with open(input_file_path, encoding=self.file_encoding) as input_file:
+        self._file_encoding = None
+        if not isinstance(input_file_path, StringIO):
```

Comment:
Why is this just StringIO and not also for ByteIO? Perhaps add a comment explaining too.

Reply:
Strings don't need decoding bc they have already been decoded.